ui/app/routes/vault/cluster/secrets/backend/configuration.js

```diff
@@ -9,40 +9,15 @@ export default Route.extend({
     if (this.wizard.featureState === 'list') {
       this.wizard.transitionFeatureMachine(this.wizard.featureState, 'CONTINUE', backend.get('type'));
     }
-    // if KV2 then we pull in specific attrs from the config endpoint saved on the secret-engine record and display them
     if (backend.isV2KV) {
-      let secretEngineRecord = this.store.peekRecord('secret-engine', backend.id);
```

Comment:
is set on the model. my bad for not figuring this out in the first PR. I really should have pulled this config work into a separate PR to start with.